doc/concepts/labels.rst

```diff
@@ -98,6 +98,16 @@ the region of cables that have radius less than 0.5 μm
 
 .. _labels-expressions:
 
+.. glossary::
+  iexpr
+    An iexpr is an inhomogeneous expression, that can be evaluated at any point on a cell.
+
+Some common inhomogeneous expression:
```

Comment:
These are somewhat redundant, since the full list follows directly.

Reply:
I tried to keep it in the same format as the description of regions and locsets above, where simple examples are given in the same way. I don't mind removing it, if you prefer less redundancy over consistency.

Reply:
I don't like redundancy, no ;)